recipes/mosaic/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "0.3.1" %}
+{% set sha256 = "9561884699ed7f56e6e5542753babc8ef0461ea71e4c125bc50ca46b147f40bb" %}
+
+package:
+  name: mosaic
+  version: {{ version }}
+
+source:
+  url: https://github.com/weber8thomas/mosaicatcher/archive/refs/tags/{{ version }}.tar.gz
+  sha256: '{{sha256}}'
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make
+    - pkg-config
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - boost-cpp
+    - htslib
+    - zlib
+    - bzip2
+
+  run:
+    - boost-cpp
+    - htslib
+    - zlib
+    - bzip2
+
+# requirements:
```

Comment:
Can you remove this if its not needed?

Reply:
I just removed all the unnecessary commented parts in the last commit